src/components/MainPage/constants.js

```diff
@@ -14,6 +14,9 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
+export const CORRECTIONS = "CORRECTIONS";
+export const JAILS = "JAILS";
+export const LS_TAB_KEY = "activeTab";
```

Comment:
this is also pre-existing but should this be "CORRECTIONS" so that it defaults to that, the same way we did for monthly vs annual?

Reply:
fixed